introduction.md

```diff
@@ -1 +1,3 @@
-This is an introduction to CodeSchool.  I like it so far. I've only complete one path, JavaScript, so far.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?